docs/user-guide.md

```diff
@@ -752,6 +753,14 @@ The Spark Operator provides limited support for resource quota enforcement using
 
 If you are running Spark applications in namespaces that are subject to resource quota constraints, consider enabling this feature to avoid driver resource starvation. Quota enforcement can be enabled with the command line arguments `-enable-resource-quota-enforcement=true`. It is recommended to also set `-webhook-fail-on-error=true`.
 
+## Running multiple instances of the operator within the same k8s cluster
+
+If you need to run multiple instances of the operator within the same k8s cluster. Therefore, you need to make sure that the running instances should not compete for the same custom resources or pods. You can achieve this:
+
+* By specifying custom labels on resources by defining for each instance of the operator a different `-label-selector-filter (e.g. env=dev,app-type=spark)` option for custom resources such as `SparkApplication`.
+* Running different `webhook` instances by specifying different `-webhook-svc-name` option for each deployment of the operator.
```

Comment:
We should also mention the `namespace` flag for specifying the namespace to watch for objects and `webhook-svc-namespace` for specifying the namespace of the webhook service resource.

Reply:
Actually, I propose 2 options (the second one is more interesting IMHO):
- We specify one single `namespace` (this is doable by specifying the flag namespace)
- Suppose we have different namespaces for environment dev, then : We add label `env:dev` to resources within these namespaces. This makes the operator listening to all namespaces + filter according to the specified labels.